### DISCUSSION TITLE
Has anyone been able to get the new "Live Update" package to work?

### DISCUSSION THREAD

**Discussion by ldiebold** [capawesome-team/capacitor-plugins#171] [Q&A]
Hey there! I noticed the new "Live Update" package is out and it looks awesome!
Just having some trouble getting it to work. Any ideas? Here's what I tried.

Call the live update function:
```ts
const downloadBundle = async () => {
  await LiveUpdate.downloadBundle({
    url: 'https://my-file.com/6.0.0.zip',
    bundleId: '6.0.0',
  })
}
```

no errors are thrown.
![image](https://github.com/capawesome-team/capacitor-plugins/assets/1305724/bbab3d88-a5b0-48f0-a026-83f3f2ce0ab1)

Try setting the bundle:
```ts
async function setBundle () {
  await LiveUpdate.setBundle({ bundleId: '6.0.0' })
}
```

It logs that the bundle cannot be found:
![image](https://github.com/capawesome-team/capacitor-plugins/assets/1305724/27a49e56-7361-433c-94b2-1bf31fb0aeee)

When logging the bundles, none show up:
```ts
async function getBundles () {
  const bundles = await LiveUpdate.getBundles()
  console.log('Bundle Ids', bundles.bundleIds) // empty array
}
```

I've tried:
- several option for the server hosting the file (http, and https with a valid certificate)
- a couple of different methods for zipping the file (Mostly just tried zipping it manually. I'm using Ubuntu with the built in archive manager)
- simplifying the zipped file to nothing more than an `index.html` file
- ensuring the assets aren't nested in a sub folder

Any help appreciated. Thankyou!

**Comment by robingenz** [comment]
Please share your Logcat/XCode logs so i can take a closer look.
